python/examples/myProtocol.py

```diff
@@ -84,7 +87,8 @@ def stopStream(self, topic):
         if self.topic == topic and self.subscribers > 0:
             self.subscribers -= 1
             if self.subscribers == 0 and self.loopTask:
-                self.loopTask.stop()
+                print('canceling loop task')
+                self.loopTask.cancel()
```

Comment:
Does that mean you have a way to cancel `schedule_callback`?

Reply:
Yes you get something back from both `schedule_callback` and `schedule_coroutine` on which you can call `cancel()`

Reply:
that is great